### PR TITLE
Add top holders balance metric

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

Return the number of tokens/coins, held by the top N addresses of a given asset.
Note that over time the top holders' addresses may change - today's top 20 could be different than yesterday's top 20.

---

Request: 
-  Must use selector object in order to change the default number of holders_count
- If `slug` is used, holders_count has a default value of 10

```graphql
{
  getMetric(metric: "top_holders_balance") {
    timeseriesData(
      selector: {slug: "santiment", holders_count: 20}
      from: "2020-01-10T00:00:00Z"
      to: "2020-03-14T00:00:00Z"
      interval: "1d") {
        datetime
        value
    }
  }
}
```

Response:
```json
{
  "data": {
    "getMetric": {
      "timeseriesData": [
        {
          "datetime": "2020-01-10T00:00:00Z",
          "value": 79134020.77889353
        },
        {
          "datetime": "2020-01-11T00:00:00Z",
          "value": 79132148.51805307
        },
        {
          "datetime": "2020-01-12T00:00:00Z",
          "value": 79131434.70721608
        },
....
```

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase/clickhouse/top_holders/metric_adapter.ex [hunk 1]
```diff
@@ -0,0 +1,184 @@
+defmodule Sanbase.Clickhouse.TopHolders.MetricAdapter do
+  @behaviour Sanbase.Metric.Behaviour
+
+  @moduledoc ~s"""
+  Uses ClickHouse to calculate the percent supply in exchanges, non exchanges and combined
```
Comment (tspenov) [inline lib/sanbase/clickhouse/top_holders/metric_adapter.ex]: leftover

File: lib/sanbase/clickhouse/top_holders/metric_adapter.ex [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+      ClickhouseRepo.query_transform(query, args, fn [timestamp] ->
+        DateTime.from_unix!(timestamp)
+      end)
+      |> maybe_unwrap_ok_value()
+    end
+  end
+
+  @impl Sanbase.Metric.Behaviour
+  def available_slugs() do
+    Sanbase.Cache.get_or_store({:slugs_with_prices, 1800}, fn ->
```
Comment (tspenov) [inline lib/sanbase/clickhouse/top_holders/metric_adapter.ex]: why `:slugs_with_prices` ?

File: lib/sanbase/clickhouse/top_holders/top_holders.ex [hunk 3]
```diff
@@ -64,7 +64,7 @@ defmodule Sanbase.Clickhouse.TopHolders do
       toUnixTimestamp(intDiv(toUInt32(toDateTime(dt)), ?6) * ?6) AS time,
       sumIf(partOfTotal, isExchange = 1) * 100 AS in_exchanges,
       sumIf(partOfTotal, isExchange = 0) * 100 AS outside_exchanges,
-      in_exchanges + outside_exchanges AS in_top_holders_total
+      in_exchanges + ah AS in_top_holders_total
```
Comment (tspenov) [inline lib/sanbase/clickhouse/top_holders/top_holders.ex]: is that by mistake ? `ah` is not present

File: lib/sanbase/clickhouse/top_holders/metric_adapter.ex (hunk 2 above)
Comment (IvanIvanoff) [reply lib/sanbase/clickhouse/top_holders/metric_adapter.ex]: copy-pasta

File: lib/sanbase/clickhouse/top_holders/top_holders.ex (hunk 3 above)
Comment (IvanIvanoff) [reply lib/sanbase/clickhouse/top_holders/top_holders.ex]: yeah, typo

**Comment by tspenov** [comment]
LGTM, though build is red

**Pull request merged**